src/api/forms/draft-form-definition-repository.test.js

```diff
@@ -9,8 +9,11 @@ import {
 import { sdkStreamMixin } from '@smithy/util-stream'
 import { mockClient } from 'aws-sdk-client-mock'
 
+import {
+  create,
+  get
+} from '~/src/api/forms/draft-form-definition-repository.js'
```

Comment:
Shall we stick to the namespace import here like in `service[.test].js`?

```suggestion
import * as draftFormDefinition from '~/src/api/forms/draft-form-definition-repository.js'
```